editor/editor_inspector.cpp

```diff
@@ -2246,7 +2246,7 @@ void EditorInspectorArray::_setup() {
 			}
 			move_vbox->add_child(ae.move_texture_rect);
 
-			if (element_position < _get_array_count() - 1) {
+			if (element_position < count - 1) {
```

Comment:
Counts are cached at the top of the function, so no need to call this again here.